packages/-ember-data/tests/unit/adapters/rest-adapter/group-records-for-find-many-test.js

```diff
@@ -48,14 +49,10 @@ module('unit/adapters/rest_adapter/group_records_for_find_many_test - DS.RESTAda
       },
     });
 
-    store = createStore({
-      adapter: GroupsAdapter,
-      testRecord: DS.Model.extend(),
-    });
-  });
+    this.owner.register('adapter:application', ApplicationAdapter);
```

Comment:
Are there any cases where we have to register the corresponding `RESTSerializer`? (previously the createStore was doing this)

Reply:
The `createStore` helper was actually registering the `-default` serializer as the `JSONAPISerializer`:

https://github.com/emberjs/data/blob/7c7ebdfb3c0d586cb52f42e57815cbea665c9331/packages/-ember-data/tests/helpers/store.js#L70

There were a few cases where I needed to do this where the test involves mocking the response from the adapter. 
Some examples:

https://github.com/emberjs/data/pull/6347/files#diff-8aaaee520d117ac876f402e7a684ebd8R34
https://github.com/emberjs/data/pull/6347/files#diff-8aaaee520d117ac876f402e7a684ebd8R101
https://github.com/emberjs/data/pull/6347/files#diff-8aaaee520d117ac876f402e7a684ebd8R189